js/calculator.js

```diff
@@ -1,47 +1,43 @@
-const slider = document.getElementById("myRange");
-const output = document.getElementById("demo");
-const checkBox = document.getElementById("checkbox");
-const discountHTML = document.getElementById("discount");
-const price = document.getElementById("cena");
-var count = 0;
+const slider = document.getElementById("myRange"),
+  output = document.getElementById("demo"),
+  checkBox = document.getElementById("checkbox"),
+  discountHTML = document.getElementById("discount"),
+  price = document.getElementById("cena");
+let count = 0;
 
 // document.querySelector(".submit").addEventListener('click', sliderChange);
 document.querySelector("#calcButton").addEventListener('click', calcDisplay);
 document.querySelector("#exitButton").addEventListener('click', calcDisplay);
 
 
 function calcDisplay() {
-    count++;
-    console.log(count);
-  if(count % 2 !=0) {
-      
+  count += 1;
```

Comment:
Change the value to the boolean. And then operate on it.

Reply:
Understand, I'll do it today, promise!